airdrop/infrastructure/models.py

```diff
@@ -145,9 +150,32 @@ class UserReward(Base, AuditClass):
     )
     address = Column("address", VARCHAR(50), nullable=False, index=True)
     condition = Column("condition", TEXT, nullable=True)
-    rewards_awarded = Column("rewards_awarded", INTEGER, nullable=False)
+    rewards_awarded = Column("rewards_awarded", BIGINT, nullable=False)
+    score = Column("score", DECIMAL(10,5), nullable=False)
+    log10_score = Column("log10_score", DECIMAL(10,8), nullable=False)    
```

Comment:
We should not have a log10_score column? We should keep a generic name like the eligible amount. This name is specific to the airdrop portal where this formula is applicable.

Reply:
Agree. Will update